config/main.py

```diff
@@ -1421,15 +1421,17 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cach
         # or by default DEFAULT_CONFIG_DB_FILE. In the case of database service running in namespace,
         # the default config_db<namespaceID>.json format is used.
 
+
         config_gen_opts = ""
+        
+        if os.path.isfile(INIT_CFG_FILE):
+            config_gen_opts += " -j {} ".format(INIT_CFG_FILE)
```

Comment:
>-j [](http://example.com/codeflow?start=33&length=2)

Offline discussed.
When `-j` appears more than once on sonic-cfggen command line,  multiple json files are combined, and the latter take priority.

It makes sense to me.